server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -236,6 +243,13 @@ private void registerMetrics() {
     jettyServer.addServlet(
         new CommonMetricsServlet(JvmMetrics.getCollectorRegistry(), true),
         "/prometheus/metrics/jvm");
+
+    metricReporter = MetricReporterFactory.getMetricReporter(shuffleServerConf, id);
```

Comment:
> If user/administrator misconfigured the metric reporter class, this would throw an exception.

Why?

Reply:
I think throw an exception is better in this case. This is a user's mistake and the user will restart the server If we just logging the error. If some partitions have been assigned to this server, it will be troublesome.